package.json

```diff
@@ -63,6 +63,12 @@
           "default": "lfortran",
           "description": "The path to the LFortran compiler executable."
         },
+        "LFortranLanguageServer.compiler.flags": {
```

Comment:
This will need to be an array since the executed command is not a concatenated string. `lfortran` is invoked by its path and the arguments for it are passed as an array. If the flags are space-separated then they will be treated as a single argument which will almost certainly cause a failure. I will submit a PR against this branch with the fix.

VSCode handles arrays in its configuration oddly. In my local feature branch, I've added the option to specify additional include paths but the added paths do not appear in the configuration UI. They /do/ appear if you edit the JSON configuration file. I will update the documentation accordingly.

Reply:
Resolved in favor of https://github.com/Pranavchiku/lfortran-lsp/pull/1